dwave_networkx/generators/zephyr.py

```diff
@@ -680,3 +688,85 @@ def source_to_inner(q):
 
     for offset in offset_list:
         yield make_mapping(source_to_inner, zephyr_to_target, offset)
+
+def zephyr_torus(m, t=4, node_list=None, edge_list=None):
+    """
+    Creates a Zephyr graph modified to allow for periodic boundary conditions and translational invariance.
+    
+    The graph matches the local connectivity properties of a standard zephyr graph,
+    but with modified periodic boundary condition. Tiles of 8t qubits are arranged
+    on an m by m torus. 
+
+    Parameters
+    ----------
+    m : int
+        Grid parameter for the Zephyr lattice.
+        Connectivity of all nodes is :math:`4t+min(2m-1,4)`
+    t : int
+        Tile parameter for the Zephyr lattice.
+    node_list : iterable, optional (default None)
+        Iterable of nodes in the graph. If None, calculated from ``m``.
+        Note that this list is used to remove nodes, so only specified nodes
+        that belong to the base node set (described in the ``coordinates``
+        parameter) are added.
```

Comment:
```suggestion
        parameter of :func:`.zephyr_graph`) are added.
```

I added this before I saw line 715 (````coordinates`` parameter.    coordinates : bool, optional (default False)``) so ignore if not intended